packages/docusaurus-preset-shiki-twoslash/docusaurus-theme-shiki-twoslash/theme/CodeBlock/index.js

```diff
@@ -13,7 +13,7 @@ const CodeBlock = ({ children, ...props }) => {
 
   const handleCopyCode = () => {
     if (pre.current) {
-      copy([...(pre.current.querySelectorAll("code div.line") ?? [])].map(el => el.textContent).join("\n"))
+      copy(Array.from(pre.current.querySelectorAll("code div.line")).map(el => el.textContent).join("\n"))
     }
     setShowCopied(true)
```

Comment:
I was wondering if we want to keep showing the "notification" popup even when it fails.

The `copy` function returns a `boolean` to let us know if the copy worked, so we could handle it differently if you prefer.

Reply:
I think it's fine. The original implementation also does the same.